### DISCUSSION TITLE
Custom field in Action modal to not show label

### DISCUSSION THREAD

**Discussion by Pamposgsk** [laravel/nova-issues#4313] [Q&A] [answered]
Hi ,
Is there a way to hide the label of a field .
Sometimes there is a need to hide the label on the left side of a custom field ,in order to have more room space.Is there a way to hide this label?
Thanks
Pampos Hadjimichael

**Comment by davidhemphill** [comment, marked as answer]
There's no built-in way to do this but you could accomplish it by not using the built-in `DefaultField` and implementing the entire layout yourself.

**Comment by Pamposgsk** [comment]
Hi,
Thanks for quick reply
So you mean i need to create a custom field?
If yes I’ve already created custom field but still showing the label.
If no ,Could you please give some more details on how to create a new layout?
Thanks

**Reply by davidhemphill** [reply]
You can't turn on the label in the default custom field setup. You must remove the `DefaultField` component reference inside your Vue component and implement the layout you desire for the field instead.

**Comment by Pamposgsk** [comment]
Great thanks!
